routes/notifications.js

```diff
@@ -0,0 +1,23 @@
+var express = require("express");
+var Q = require("q");
+var marked = require("marked");
+var Sequelize = require("sequelize");
+var permissions = require("../permissions");
+var Activity = require("../models/activity");
+var Group = require("../models/group");
+var User = require("../models/user");
+
+var router = express.Router();
+
+router.route("/portraitRight/:id")
+    .put(function (req, res) {
+        var userId = res.locals.session ? res.locals.session.user : null;
```

Comment:
Comments here would be appreciated. (the code is not too complicated, but still)